recipes/zulu-openjdk/all/test_package/conanfile.py

```diff
@@ -1,16 +1,24 @@
-from conans import ConanFile
-from six import StringIO
+from conans import ConanFile, tools
+from io import StringIO
 
 
 class TestPackage(ConanFile):
+    settings = "os", "arch"
 
     def build(self):
         pass # nothing to build, but tests should not warn
 
     def test(self):
+        if tools.cross_building(self):
```

Comment:
I am fine with this. Just for clarification, the usual pattern you find in conan-center-index recipes is
```
if not tool.cross_building(self):
    ....
```

Reply:
OK, thanks! If some one says I shall change it, I will